style.css

```diff
@@ -0,0 +1,78 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: "Rubik", sans-serif;
+  line-height: 1;
+  font-weight: 400;
+  width: 375px;
+  margin: 0 auto;
+}
+
+.gen-padding {
+  padding-right: 2.4rem !important;
+  padding-left: 2.4rem !important;
+}
+
```

Comment:
- Hey, according to the list of microverse [HTML & CSS requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), You should not overuse or use `!important` except you're using a CSS framework that has it applied to styles. let's try to add the styles without making use of it.